paddlecloud/paddlejob/views.py

```diff
@@ -293,6 +293,16 @@ def get(self, request, format=None):
             .list_namespaced_resource_quota(namespace)
         return Response(quota_list.to_dict())
 
+class GetUserView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, format=None):
+        content = {
+            'user': unicode(request.user.username),  # `django.contrib.auth.User` instance.
```

Comment:
We use user's email as username all the time, and emails are strictly checked. Also using django's model values will return unicode typed strings by default.

So don't convert the type.